code/src/Components/TodoCounter.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { useSelector } from "react-redux";
+
+
+const TodoCounter = () => {
+
+    const todo = useSelector((store) => store.todo.items);
+    const remainingTodos = todo.filter((todo) => todo.done === false)
+
+    if (todo.length > 0) {
+        return (
+            <section className="counter-total">
+                <h2>You have {remainingTodos.length} todo's! </h2>
+            </section>
+        );
+  } else if (remainingTodos.length === 0) {
```

Comment:
Could also write else instead of else if and exclude the comparator task, but I guess one benefit of writing it as you did here is simplified readability, which is nice!